File: FizzBuzz Solutions/fizzbuzz.py
def fizzbuzz(n):
    """Implement the FizzBuzz algorithm."""
    for i in range(1, n + 1):
        if i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz")
        elif i % 3 == 0:
            print("Fizz")
        elif i % 5 == 0:
            print("Buzz")
        else:
            print(i)

def main():
    """Main function to execute the FizzBuzz algorithm."""
    fizzbuzz(100)

if __name__ == "__main__":
    main()

File: FizzBuzz Solutions/fizzbuzz.R
fizzBuzz <- function(n) {
  # Implement the FizzBuzz algorithm
  for (i in 1:n) {
    if (i%%3==0 & i%%5==0) {
    print("FizzBuzz")
    } else if (i%%3==0) {
    print("Fizz")
    } else if (i%%5==0) {
    print("Buzz")
    } else {
    print(i)
    }
  }
}

main <- function() {
  # Main function to execute the FizzBuzz algorithm
  fizzBuzz(100)
}

main()